Ryoung27/redditClone
# Ry reddit day one two

steps to test
pull down code

git fetch --all
git checkout rm-user-endpoint
make migrations
python manage.py makemigrations
python manage.py migrate
start server with python manage.py runserver
Check browsable API and make sure a post was created.
